packages/ts-logger/src/logger.ts

```diff
@@ -75,7 +75,7 @@ export function createLogger(options: LoggerOptions | undefined = {}): Logger {
   };
 
   const enchancedDefaultMeta = {
-    releaseSha: process.env.RELEASE_SHA || null,
+    releaseSha: process.env.RELEASE_SHA || process.env.GIT_SHA || null,
```

Comment:
Sometimes the sha is RELEASE_SHA (ECS, DOCKER), sometimes its GIT_SHA, for now lets try both and we can clean this up in future prs.